script.js

```diff
@@ -0,0 +1,193 @@
+/* eslint-disable no-undef */
+const getDiv = (() => document.createElement('div'));
```

Comment:
Sounds like a getter function, but it is actually a factory function. We can name it as createDiv, createLabel.